controllers/discovery/initDiscoveryTopicData.js

```diff
@@ -42,11 +51,21 @@ const InitDiscoveryTopicData = async (req, res) => {
     });
     let suggestedTopics = topicWithCommonFollowerResult;
 
+    let totalData = await sequelize.query(totalDataQuery, {
+      type: QueryTypes.SELECT,
+      replacements: {
+        userId
+      },
+      raw: true
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code correctly executes the `totalDataQuery` and retrieves the total data count.



<!-- This is an auto-generated comment by OSS CodeRabbit -->